math.md

```diff
@@ -952,14 +903,16 @@ type Svg struct {
 ```
 
 We could make adjustments to this if we needed to (like changing the name of the
-struct to `SVG`) but it's definitely good enough to start us off.
+struct to `SVG`) but it's definitely good enough to start us off. Paste that struct into the same file as the clockface functions.
```

Comment:
Actually, looks like you wanted this in the test file, so I should edit this! TODO.